### PR TITLE
[WIP/Outdated] mk_sdcard_image changes

### PR DESCRIPTION
**Pull request by arminn**

### CODE REVIEW

File: doc/mk_sdcard_image.sh [hunk 1]
```diff
@@ -26,6 +31,7 @@ usage()
 	echo "  -s image-size   Optional, image size in GiB"
 	echo "  -u domain       Optional, unpack only specified domain: dom0, domd, domf, doma, domu"
 	echo "  -f              Optional, force rewrite of image file (useful for batch usage)"
+	echo "  -r              Optional, force repack of initiramfs and add DomA Image(from DomA dir) to it"
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: Not sure we want this patch at all: this looks like a mixed functionality.
Even more, we do not ship the helper script with the builds.
at the end of the day, why DomA only?

File: doc/mk_sdcard_image.sh [hunk 2]
```diff
@@ -34,6 +37,7 @@ usage()
 	echo "  -u domain       Optional, unpack only specified domain: dom0, domd, domf, doma, domu"
 	echo "  -f              Optional, force rewrite of image file (useful for batch usage)"
 	echo "  -r              Optional, force repack of initiramfs and add DomA Image(from DomA dir) to it"
+	echo "  -v              Optional, create specific android config with Android verified boot cmdline"
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: IMO, mixed functionality. The original intention of this script to flash images as they are without altering

File: doc/mk_sdcard_image.sh (hunk 1 above)
Comment (arminn) [reply doc/mk_sdcard_image.sh]: From my point of view: mk_sdk_image should provide the functionality to create a final image using build artifacts. For someone is very convenient to build DomA, and use product builds with modified DomA. As you know, due to Xen design we do not use boot.img in DomA, we use Image placed into Initramfs. Without this change, DomA integration is incomplete, despite Image is copied in to out artifacts.

**Comment by otyshchenko1** [comment]
ping, this patch should be either updated and merged (if there is a need) or just dropped.

**Comment by arminn** [comment]
It will be changed someday.

**Comment by rshym** [comment]
As for me two commits can be merged:
"Remove DomA hardcoded partition id's"
"Added metadata,vbmeta partitions for DomA"

One commit can't be merged
"Get rid of mebibytes and gibibytes"
has mistakes in calculation, if we divide by 1024 we have MiB/GiB, not MB/GB.

**Pull request closed without merging**